src/Symfony/Component/ExpressionLanguage/Node/GetAttrNode.php

```diff
@@ -69,6 +69,10 @@ public function evaluate(array $functions, array $values)
     {
         switch ($this->attributes['type']) {
             case self::PROPERTY_CALL:
+                if ($this->nodes['node'] instanceof GetAttrNode) {
+                    $this->nodes['node']->attributes['is_null_coalesce'] = $this->attributes['is_null_coalesce'];
```

Comment:
shouldn't this be `$this->nodes['node']->attributes['is_null_coalesce'] = $this->nodes['node']->attributes['is_null_coalesce'] || $this->attributes['is_null_coalesce'];` ?

Reply:
Well, at the beginning I wrote it like that then I found it not easy to read due to the complex structure required to retreive the tested data. Eitherway is fine as long as it's readable.

Reply:
This will introduce a regression when chained with other types of nodes (other than the GetAttrNode).

Reply:
This solution seems to be for null-coalescing, which is different than null-safe.

I have a different solution, or at least an additional one. Modify lines 88-89 to:

```php
if ($this->attributes['is_null_coalesce'] || $this->nodes['attribute']->isNullSafe) {
    return $obj->$property ?? null;
}
```

This seems to fix the issue where `x?.y?.z` and `x.y` does not exist.

Reply:
I've created a more complete patch: https://github.com/symfony/symfony/pull/49228

Reply:
```suggestion
                    $this->nodes['node']->attributes['is_null_coalesce'] = $this->attributes['is_null_coalesce'] ?? false;
```